docs/api/api.yml

```diff
@@ -1059,12 +1059,15 @@ components:
       properties:
         scheme:
           type: string
+          description: "Possible values: degrees, RD."
         x:
           type: string
           format: "number"
+          description: "In case of RD: range from -7000 to 300000. In case of degrees: latitude, range from -90 to 90."
         y:
           type: string
           format: "number"
+          description: "In case of RD: range from 289000 to 629000. In case of degrees: longitude, range from -180 to 180."
```

Comment:
When simplifying the valid area for RD to a rectangle, we have an area of 11x11 km of overlap between possible x/y values. However, the actual [valid area](https://nl.wikipedia.org/wiki/Rijksdriehoeksco%C3%B6rdinaten#Geldigheidsgebied) is a polygon.

Reply:
@lindareijnhoudt do we want to improve the validation of the RD coordinates, given the rules explained in the link above. If we do, we need to do that in all services (`easy-deposit-ui`, `easy-split-multi-deposit`, `easy-validate-dans-bag` (???), `easy-ddm` (???)).

Reply:
I'll merge this PR for now, though this question has not yet been resolved. But if we want this more precise validation, we'll make a different issue for that anyway.

Reply:
it makes sense to have it in easy-deposit-ui and in split-multi-deposit, as that are entry points. for the sword-input we make the assumption the front-office takes care of it. 
doesn't the opengis gml already validate this? (i guess not the RD)